helm/cluster-openstack/files/etc/ssh/sshd_config

```diff
@@ -0,0 +1,15 @@
+# Use most defaults for sshd configuration.
+Subsystem sftp internal-sftp
+ClientAliveInterval 180
+UseDNS no
+UsePAM yes
+PrintLastLog no # handled by PAM
+PrintMotd no # handled by PAM
+# Non defaults (#100)
+ClientAliveCountMax 2
+PasswordAuthentication no
+TrustedUserCAKeys /etc/ssh/trusted-user-ca-keys.pem
+MaxAuthTries 5
+LoginGraceTime 60
+AllowTcpForwarding no
+AllowAgentForwarding no
```

Comment:
I moved that from the inline `content:` only. No changes to the SSH config.

Reply:
Nice improvement.